JavaAssignment/src/ravi/ClientSimpleInterest.java

```diff
@@ -0,0 +1,9 @@
+package ravi;
+
+public class ClientSimpleInterest {
+
+	public static void main(String[] args) {
```

Comment:
Same comments are applicable for here, Need to add comments before every methods & class.